docker-compose.yml

```diff
@@ -19,6 +19,7 @@ services:
       - DB_USERNAME=my_user
       - DB_DATABASE=thatsmybis_1
       - DB_PASSWORD=my_password
+      - COMPOSER_MEMORY_LIMIT=-1
```

Comment:
Fun times with `composer install`.

Reply:
Is this something that you'd like to see differently btw? Not sure how to interpret. The docker setup didn't work for me out of the box btw which is why I added this. I'm not using it actively atm since I have my own setup, but thought I'd at least test it.

Reply:
Nah it's probably fine. I don't use docker, but I do run into issues where `composer` runs out of memory and this option needs to be passed in via the CLI.